modules/swagger-codegen/src/main/resources/LoopBackSwift/model.mustache

```diff
@@ -0,0 +1,46 @@
+{{#models}}{{#model}}import ObjectMapper
+{{#useRealm}}
+import Realm
+import RealmSwift
+{{/useRealm}}
+
+public class {{classname}} : {{#useRealm}}Object, {{/useRealm}}Mappable, CustomStringConvertible {
```

Comment:
Each template should have a comment briefly describing what the generated method will do. You should also add a comment for things that are not obvious (like conditional generation of code for handling auth, etc.). If there is a particularly difficult to read long string of tokens, you should add a comment briefly explaining what's happening there or some example output code. The api.mustache template is a good example - it's very difficult to image the code in its final form so it would be handy to have a example output of the template in the comments above the code. The goal is for someone not familiar with the code to easily see what's going on (If it's obvious, then no need for a comment).